src/main/java/org/bytedeco/javacv/Java2DFrameConverter.java

```diff
@@ -107,25 +107,25 @@ public static int decodeGamma22(int value) {
     public static int encodeGamma22(int value) {
         return gamma22inv[value & 0xFF] & 0xFF;
     }
-    public static void flipCopyWithGamma(ByteBuffer srcBuf, int srcStep,
-            ByteBuffer dstBuf, int dstStep, boolean signed, double gamma, boolean flip, int channels) {
+    public static void flipCopyWithGamma(ByteBuffer srcBuf, int buffer_index, int srcStep,
+                                         ByteBuffer dstBuf, int dst_buffer_index, int dstStep, boolean signed, double gamma, boolean flip, int channels) {
```

Comment:
Let's keep using camel case for the new variables.

Reply:
There's still a few more left, please update!

Reply:
Oh right, didn't noticed them ! 2min..